nba-pipeline/README.md

```diff
@@ -65,22 +63,22 @@ zenml integration install kubeflow -f
 
 ## :basketball: The Task
 
-A couple of weeks ago, we were looking for a fun project to work on for the next chapter of our ZenHacks. During our 
-initial discussions, we realized that it would be really great to work with an NBA dataset, as we could quickly get 
-close to a real-life application like a "3-Pointer Predictor" while simultaneously entertaining ourselves with one 
+A couple of weeks ago, we were looking for a fun project to work on for the next chapter of our ZenHacks. During our
+initial discussions, we realized that it would be really great to work with an NBA dataset, as we could quickly get
+close to a real-life application like a "3-Pointer Predictor" while simultaneously entertaining ourselves with one
 of the trending topics within our team.
 
 As we were building the dataset around a "3-Pointer Predictor", we realized that there is one factor that we need to
-take into consideration first: Stephen Curry, The Baby Faced Assassin. In our opinion, there is no denying that he 
+take into consideration first: Stephen Curry, The Baby Faced Assassin. In our opinion, there is no denying that he
 changed the way that the games are played in the NBA and we wanted to actually prove that this was the case first.
 
-That's why our story in this ZenHack will start with a pipeline dedicated to drift detection. As the breakpoint of this 
-drift, we will be using the famous "Double Bang" game that the Golden State Warriors played against Oklahoma City 
-Thunder back in 2016. Following that, we will build a training pipeline which will generate a model that predicts 
-the number of three-pointers made by a team in a single game, and ultimately, we will use these trained models and 
+That's why our story in this ZenHack will start with a pipeline dedicated to drift detection. As the breakpoint of this
+drift, we will be using the famous "Double Bang" game that the Golden State Warriors played against Oklahoma City
+Thunder back in 2016. Following that, we will build a training pipeline which will generate a model that predicts
+the number of three-pointers made by a team in a single game, and ultimately, we will use these trained models and
 create an inference pipeline for the upcoming matches in the NBA.
 
-![Diagram depicting the Training and Inference pipelines](_assets/Training and Inference Pipeline.png)
+![Diagram depicting the Training and Inference pipelines](\_assets/Training and Inference Pipeline.png)
```

Comment:
What is this backslash doing here? Is this a Windows thing?

Reply:
yeah removed it :)